x11-misc/util_macros/util_macros-1.19.2.recipe

```diff
@@ -35,7 +35,7 @@ BUILD_PREREQUIRES="
 
 BUILD()
 {
-	#./autogen.sh -V
+	autoreconf -vfi
```

Comment:
Is this necessary?

Reply:
As there is no "autogen.sh" file and the source provides a configure.ac I thought it would be useful?

Reply:
The source already have a `configure`, so it's not necessary unless you need to modify `configure`

Reply:
Not needed it seems :)